examples/platforms/posix/radio.c

```diff
@@ -308,16 +309,24 @@ void otPlatRadioSetPromiscuous(bool aEnable)
 void posixRadioInit(void)
 {
     struct sockaddr_in sockaddr;
+    char *offset;
     memset(&sockaddr, 0, sizeof(sockaddr));
     sockaddr.sin_family = AF_INET;
 
+    offset = getenv("PORT_OFFSET");
+
+    if (offset)
+    {
+        sPortOffset = atoi(offset) * WELLKNOWN_NODE_ID;
```

Comment:
I generally favor `strtoul` instead of `atoi` because of better error handling and default parsing of decimal and hex numbers